tests/README.md

```diff
@@ -0,0 +1,22 @@
+# Running tests
+
+Installing and running tests
+```bash
+pip install -r requirements
+pytest
+```
+
+Running tests with gpu
+```bash
+pytest --gpu
+```
+
+Run tests without re-downloading models
+```bash
+pytest --model-cache ./cache
+```
+
+All subsequent runs could be made using
+```bash
+pytest --model-cache ./cache --skip-download
```

Comment:
tests should be executed the same way for the first run and subsequent as well. model_download.py should detect the the models are locally.